crates/database/Cargo.toml

```diff
@@ -40,3 +40,9 @@ rand = "0.8"
 dashmap.workspace = true
 thiserror.workspace = true
 tracing.workspace = true
+tokio-postgres-rustls.workspace = true
+rustls.workspace = true
+rustls-native-certs.workspace = true
+rustls-pemfile = "2.2.0"
+rustls-pki-types = "1.10.0"
+
```

Comment:
These can be inherited from the workspace as well. Also.. are they needed?

Reply:
no, rm'd